src/main/scala/viper/gobra/frontend/InformativeErrorListener.scala

```diff
@@ -20,6 +20,19 @@ import scala.collection.mutable.ListBuffer
 
 class InformativeErrorListener(val messages: ListBuffer[ParserError], val source: Source) extends BaseErrorListener {
 
+  /**
+    * First token that Gobra introduces that is not present in Go.
+    * First refers to the order of token declarations in GobraLexer.g4
+    * as this order is preserved by ANTLR4 when generating GobraParser.java
+    */
+  private val FIRST_GOBRA_TOKEN = GobraParser.TRUE
+  /**
+    * Last token that Gobra introduces that is not present in Go.
+    * Last refers to the order of token declarations in GobraLexer.g4
+    * this order is preserved by ANTLR4 when generating GobraParser.java
+    */
+  private val LAST_GOBRA_TOKEN = GobraParser.BACKEND
+
```

Comment:
Is it guaranteed that the ids of the tokens are monotonic with respect to the position in which the tokens are declared?

Reply:
As discussed offline, we leave it as is